src/item.cpp

```diff
@@ -375,3 +468,12 @@ bool Item::operator<(const Item &rhs) const {
     std::string rhs_name = rhs.PrettyName();
     return std::tie(name,uid_,hash_) < std::tie(rhs_name, rhs.uid_, hash_);
 }
+
+bool Item::Wearable() {
```

Comment:
should be `bool Item::Wearable() const`

Reply:
I admit I'm still rusty as hell at C++, such review details are appreciated.